02-express-tutorial/app.js

```diff
@@ -1 +1,65 @@
-console.log('Express Tutorial')
+console.log("Express Tutorial");
+
+const express = require(`express`);
+// const path = require(`path`);
+const app = express();
+const { products } = require("./data");
+
+app.use(express.static("./public"));
+
+app.get(`/api/v1/test`, (req, res) => {
+  return res.json({ message: "It worked!" });
+});
+
+app.get(`/api/v1/products`, (req, res) => {
+  return res.json({ products });
+});
+
+app.get(`/api/v1/products/sortPriceLow`, (req, res) => {
```

Comment:
Rather than specifying a different method for each type of sort operation, it’d be better / more extensible to include all operations under a single query operation that accepts a bunch of query parameters. For sorts, you can have users supply a query string parameter like “sort=price” for increasing and “sort=-price” for decreasing order